src/languages/c.js

```diff
@@ -245,15 +286,15 @@ export default function(hljs) {
         { // containers: ie, `vector <int> rooms (9);`
           begin: '\\b(deque|list|queue|priority_queue|pair|stack|vector|map|set|bitset|multiset|multimap|unordered_map|unordered_set|unordered_multiset|unordered_multimap|array)\\s*<',
```

Comment:
I guess this is beyond the scope of this PR but these are all C++ types (and the entire rule related to a C++ only feature, templates).